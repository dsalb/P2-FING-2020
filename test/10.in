hacerVecinos 2 5 30
hacerVecinos 8 5 20
hacerVecinos 1 4 10
hacerVecinos 5 4 10
hacerVecinos 10 4 90
hacerVecinos 8 10 10
hacerVecinos 3 6 40
hacerVecinos 6 9 40
hacerVecinos 3 9 40
longitudesCaminosMasCortos 1
longitudesCaminosMasCortos 2
longitudesCaminosMasCortos 3
longitudesCaminosMasCortos 4
longitudesCaminosMasCortos 5
longitudesCaminosMasCortos 6
longitudesCaminosMasCortos 7
longitudesCaminosMasCortos 8
longitudesCaminosMasCortos 9
longitudesCaminosMasCortos 10
Fin
