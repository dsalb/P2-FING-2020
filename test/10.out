1>Nuevos vecinos: 2 5 30.00.
2>Nuevos vecinos: 8 5 20.00.
3>Nuevos vecinos: 1 4 10.00.
4>Nuevos vecinos: 5 4 10.00.
5>Nuevos vecinos: 10 4 90.00.
6>Nuevos vecinos: 8 10 10.00.
7>Nuevos vecinos: 3 6 40.00.
8>Nuevos vecinos: 6 9 40.00.
9>Nuevos vecinos: 3 9 40.00.
10>0.00 50.00 INF 10.00 20.00 INF INF 40.00 INF 50.00 
11>50.00 0.00 INF 40.00 30.00 INF INF 50.00 INF 60.00 
12>INF INF 0.00 INF INF 40.00 INF INF 40.00 INF 
13>10.00 40.00 INF 0.00 10.00 INF INF 30.00 INF 40.00 
14>20.00 30.00 INF 10.00 0.00 INF INF 20.00 INF 30.00 
15>INF INF 40.00 INF INF 0.00 INF INF 40.00 INF 
16>INF INF INF INF INF INF 0.00 INF INF INF 
17>40.00 50.00 INF 30.00 20.00 INF INF 0.00 INF 10.00 
18>INF INF 40.00 INF INF 40.00 INF INF 0.00 INF 
19>50.00 60.00 INF 40.00 30.00 INF INF 10.00 INF 0.00 
20>Fin.
